front/app/modules/commercial/analytics/admin/components/VisitorsCard/Chart/generateEmptyData.ts

```diff
@@ -1,26 +1,24 @@
-import moment from 'moment';
-
-// utils
-import { range } from 'lodash-es';
-
-// typings
 import {
-  TimeSeries,
-  TimeSeriesRow,
-} from '../../../hooks/useVisitorsData/typings';
+  dateRange,
+  getEmptyRow,
+} from '../../../hooks/useVisitorsData/parse/utils';
+// typings
+import moment, { Moment } from 'moment';
+import { IResolution } from 'components/admin/ResolutionControl';
 
-export const generateEmptyData = (): TimeSeries => {
-  return range(-6, 1).map((i) => {
-    const firstDayOfMonth = moment()
-      .subtract({ months: Math.abs(i) })
-      .format('YYYY-MM-01');
```

Comment:
This way, the empty data view responds to date range and resolution changes- not super essential but just a nice touch